tests/phpunit/tests/test-ajax-on-front.php

```diff
@@ -28,26 +30,81 @@ public function _ajax_test_locale() {
 		wp_die( wp_json_encode( __( 'Dashboard' ) ) ); // phpcs:ignore WordPress.WP.I18n.MissingArgDomain
```

Comment:
```suggestion
		echo wp_json_encode( __( 'Dashboard' ) ); // phpcs:ignore WordPress.WP.I18n.MissingArgDomain
		wp_die();
```

According to the WordPress test suite documentation, `WPAjaxDieStopException` should be used for error and response without output. See: https://github.com/WordPress/wordpress-develop/blob/6.0.2/tests/phpunit/includes/exceptions.php#L12-L34
So maybe we should `echo` the response and call `wp_die()` right after. This way `$this->_handleAjax()` will throw a `WPAjaxDieContinueException`. Then we will be able to use `$this->_last_response` to do our tests.

Reply:
Interesting. I never paid attention to this.
In fact I believe there is even better: https://developer.wordpress.org/reference/functions/wp_send_json/

Reply:
Indeed, it's even better!